sql/conseil.sql

```diff
@@ -448,7 +450,9 @@ CREATE TABLE tezos.operations (
     utc_time character varying NOT NULL
 );
 
-
+ CREATE INDEX ix_manager_pubkey ON tezos.operations USING btree (manager_pubkey);
+ CREATE INDEX ix_operation_group_hash ON tezos.operations USING btree (operation_group_hash);
+ CREATE INDEX ix_originated_contracts ON tezos.operations USING btree (originated_contracts);
```

Comment:
I'm honestly wondering how the new indexes don't reflects on the PlatformDiscoveryOperations, that should mention those fields as UniqueKeys... just the same as with the other similar PR targeting the release tag branch...?

Yet the tests passes...

Reply:
🤷‍♂️

Reply:
Man! I think I've commented the wrong PR, that's why I got inconsistent findings... 🤦